inginious/agent/mcq_agent/__init__.py

```diff
@@ -75,7 +75,10 @@ def environments(self):
             text.append("\n\n" + _("Among them, you have {} invalid answers in the multiple choice questions").format(mcq_error_count))
 
         nb_subproblems = len(task.get_problems())
-        grade = 100.0 * float(nb_subproblems - error_count) / float(nb_subproblems)
+        if nb_subproblems == 0:
+            grade = 100.0
```

Comment:
If I understand well, the MCQ agent will return a score of 100 if there are no problem added. Wouldn't this be rather considered as an error ?